source/upgrade.cpp

```diff
@@ -332,26 +320,20 @@ bool upgradeApplicationFromStorage(void)
                                ARM_UC_GUID_SIZE);
                     } else {
                         /* Firmware candidate size too large */
-                        tr_error("Slot %" PRIu32 " firmware size too large %"
-                                 PRIu32 " > %" PRIu32, index,
-                                 (uint32_t) imageDetails.size,
-                                 (uint32_t) MBED_CONF_MBED_BOOTLOADER_MAX_APPLICATION_SIZE);
+                    	boot_debug("[DBG ] Slot firmware size too large\r\n");
                     }
                 } else {
                     /* Integrity check failed */
-                    tr_error("Slot %" PRIu32 " firmware integrity check failed",
-                             index);
+                	boot_debug("[DBG ] Slot firmware integrity check failed\r\n");
                 }
             } else {
-                tr_info("Slot %" PRIu32 " firmware is of older date",
-                        index);
+            	boot_debug("[DBG ] Firmware is of older date\r\n");
                 /* do not print HMAC version
                 printSHA256(imageDetails.hash);
                 */
-                tr_info("Version: %" PRIu64, imageDetails.version);
             }
         } else {
-            tr_info("Slot %" PRIu32 " is empty", index);
+        	boot_debug("[DBG ] Slot is empty\r\n");
```

Comment:
This seems to be intended with TAB instead of space. Please check the remaining lines as well.

Reply:
Fixed